releases.md

```diff
@@ -36,6 +36,13 @@ Further documentation available:
 
 ## Releases
 
+### v0.28
+
+- **Latest Release**: [v0.28.0][v0-28-0] (2022-11-25) ([docs][v0-28-0-docs])
+- **Initial Release**: [v0.28.0][v0-28-0] (2022-11-25)
+- **End of Life**: 2023-01-24
```

Comment:
this should be four months

Reply:
Do we want to mark 0.28 as an LTS ?

Reply:
I guess we can start marking LTS releases from January so that they are aligned with Pipelines' LTS releases